caravel/static/default.js

```diff
@@ -0,0 +1,14 @@
+var mq = window.matchMedia('all and (max-width: 700px)');
```

Comment:
This is not necessary. This branch is not ready to merge. I just wanted to show you the UI.